.circleci/config.yml

```diff
@@ -0,0 +1,47 @@
+version: 2
+jobs:
+  build-python-common: &common-build
+    docker:
+      - image: python
+    working_directory: ~/ckantoolkit
+    steps:
+      - checkout
+      - run:
+          name: Python Version
+          command: python --version
+      - run:
+          name: Install Dependencies
+          command: pip install -r requirements.txt
+      - run:
+          name: Test Old Ckan
+          command: nosetests tests/test_old_ckan.py
+      - run:
+          name: Test New Ckan
+          command: nosetests tests/test_new_ckan.py
+  build-python27:
+    <<: *common-build
+    docker:
+      - image: python:2.7.17
```

Comment:
Should we use the minor version tags (eg `2.7`, `3.6`, `3.7`, etc) rather than the patch ones? Otherwise we'll have to remember to come back to this file when there are new releases

Reply:
Sure I will change to the minor version tags.